### DISCUSSION TITLE
blog/rails-credentials-vscode-edit

### DISCUSSION THREAD

**Discussion by giscus** [harrison-broadbent/railsnotes-comments#8] [RailsNotes Comments]
# blog/rails-credentials-vscode-edit

This short guide shows you how to use VS Code to edit secrets in your Ruby on Rails app when you run `rails credentials:edit`. I also include a handy `bin/credentials:edit` script to simplify things.

https://railsnotes.xyz/blog/rails-credentials-vscode-edit

**Comment by bryankimani** [comment]
Thank you for the handy script at the end. I will be using it in my projects.

It will save me the hustle of always typing the full `EDITOR="code --wait" rails credentials:edit` command every time I want to edit a credentials file.

**Reply by harrison-broadbent** [reply]
Thanks Bryan! Give it a go and let me know what you think — I'm keen for any feedback on it!
